__manifest__.py

```diff
@@ -21,7 +21,9 @@
         'project', # recurrence
         'stock',  # qty_available
         'website_sale',  # public_categ_ids
-        'partner_commission'],
+        'partner_commission',
+        'mrp',
```

Comment:
Because of https://github.com/sbuhl/gse_custo/commit/28058937d8740b740cf5e669143992e355fc2bfb#diff-16f1646f9b02db5a3b290a6fb333a7fcc74d04d8619c3175011dfeca42829c9aR3